static-site/src/sections/individual-group-page.jsx

```diff
@@ -0,0 +1,211 @@
+  );
+}
+
+const OverviewContainer = styled.div`
+  text-align: justify;
+  font-size: 16px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  font-weight: 300;
+  color: var(--darkGrey);
+  line-height: 1.42857143;
+  margin: 0px auto 0px auto;
+  max-width: 900px;
+`;
+
+class Index extends React.Component {
+  constructor(props) {
+    super(props);
+    configureAnchors({ offset: -10 });
+    this.state = {
+      dataLoaded: false,
+      errorFetchingData: false,
+      groupNotFound: false,
+      groupName: undefined,
+      sourceInfo: undefined
+    };
+  }
+
+  // parse getAvailable listing into one that dataset-select component accepts
+  createDatasetListing = (list, groupName) => {
+    return list.map((d) => {
+      return {
+        filename: d.request.replace(`groups/${groupName}/`, '').replace('narratives/', ''),
+        url: `https://nextstrain.org/${d.request}`,
+        contributor: groupName
+      };
+    });
+  };
+
+  async componentDidMount() {
+    const groupName = this.props["groupName"];
+    fetchAndParseJSON(`/charon/getSourceInfo?prefix=/groups/${groupName}/`)
+      .then((sourceInfo) => {
+        this.setState({sourceInfo});
+        return sourceInfo;
+      })
+      .catch((err) => {
+        console.error("Cannot find group.", err.message);
+        this.setState({groupName, groupNotFound: true});
+      })
+      .then((sourceInfo) => {
+        if (sourceInfo && (sourceInfo.showDatasets || sourceInfo.showNarratives)) {
+          return fetchAndParseJSON(`/charon/getAvailable?prefix=/groups/${groupName}/`);
+        }
+        return undefined;
+      })
+      .then((availableDataJson) => {
+        if (availableDataJson) {
+          this.setState({
+            datasets: this.createDatasetListing(availableDataJson.datasets, groupName),
+            narratives: this.createDatasetListing(availableDataJson.narratives, groupName),
+            dataLoaded: true,
+            groupName
+          });
+        }
+      })
+      .catch((err) => {
+        console.error("Error fetching / parsing data.", err.message);
+        this.setState({errorFetchingData: true});
+      });
+  }
+
+  render() {
+    return (
+      <GenericPage location={this.props.location}>
+        {this.state.sourceInfo &&
+        <>
+          <FlexCenter>
+            <Title avatarSrc={this.state.sourceInfo.avatar}>
+              {this.state.sourceInfo.title}
+              <Byline>{this.state.sourceInfo.byline}</Byline>
+              <Website>{this.state.sourceInfo.website}</Website>
+            </Title>
+          </FlexCenter>
+          {this.state.sourceInfo.overview &&
+            <FlexCenter>
+              <OverviewContainer>
+                {this.state.sourceInfo.overview}
+              </OverviewContainer>
+            </FlexCenter>
+          }
+        </>}
+        <HugeSpacer />
+        {this.state.dataLoaded && this.state.sourceInfo && this.state.sourceInfo.showDatasets && this.state.datasets.length > 0 && (
```

Comment:
In the case of a successful `getAvailable` API call, but zero datasets, we wish to display a single line such as "No datasets are present in this group". Ditto for narratives.